BACKERS.md

```diff
@@ -0,0 +1,87 @@
+# Sponsors & Backers
+
+Material-UI is an MIT-licensed open source project. It's an independent project with ongoing development made possible thanks to the support by these awesome backers. If you'd like to join them, please consider:
```

Comment:
one more tweak here:

`thanks to the support by these awesome backers`

should be:

`thanks to the support of these awesome backers`